src/libraries/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -577,15 +596,13 @@ private bool AcquireClientCredentials(ref byte[]? thumbPrint)
                 {
                     if (NetEventSource.Log.IsEnabled())
                         NetEventSource.Log.UsingCachedCredential(this);
-
```

Comment:
This blank line is valueable after the (improperly) unbraced if.

Reply:
I put it back as it was.  this seems common pattern and if anything we should do separate style PR.